src/test/resources/cohortcharacterization/CohortCharacterization.json

```diff
@@ -0,0 +1,154 @@
+{
+  "cohortExpressions": [
+    {
+      "PrimaryCriteria": {
+        "CriteriaList": [
+          {
+            "DrugExposure": {
+              "CodesetId": 0,
+              "DrugTypeExclude": false
+            }
+          }
+        ],
+        "ObservationWindow": {
+          "PriorDays": 0,
+          "PostDays": 0
+        },
+        "PrimaryCriteriaLimit": {
+          "Type": "First"
+        }
+      },
+      "ConceptSets": [
+        {
+          "id": 0,
+          "name": "clopidogrel",
+          "expression": {
+            "items": [
+              {
+                "concept": {
+                  "CONCEPT_ID": 1322184,
+                  "CONCEPT_NAME": "clopidogrel",
+                  "STANDARD_CONCEPT": "S",
+                  "STANDARD_CONCEPT_CAPTION": "Standard",
+                  "INVALID_REASON": "V",
+                  "INVALID_REASON_CAPTION": "Valid",
+                  "CONCEPT_CODE": "32968",
+                  "DOMAIN_ID": "Drug",
+                  "VOCABULARY_ID": "RxNorm",
+                  "CONCEPT_CLASS_ID": "Ingredient"
+                },
+                "isExcluded": false,
+                "includeDescendants": true,
+                "includeMapped": false
+              }
+            ]
+          }
+        }
+      ],
+      "QualifiedLimit": {
+        "Type": "First"
+      },
+      "ExpressionLimit": {
+        "Type": "First"
+      },
+      "InclusionRules": [ ],
+      "CensoringCriteria": [ ],
+      "CollapseSettings": {
+        "CollapseType": "ERA",
+        "EraPad": 0
+      },
+      "CensorWindow": { }
+    }
+  ],
+  "featureAnalyses": [
+    {
+      "id": 1,
+      "type": "PRESET",
+      "name": "Demographics gender (preset)",
+      "domain": "DEMOGRAPHICS",
+      "descr": "Gender of the subject.",
+      "design": "DemographicsGender"
+    },
+    {
+      "id": 2,
+      "type": "CRITERIA_SET",
+      "name": "Demographics gender (via criteria)",
+      "domain": "DEMOGRAPHICS",
+      "descr": "Gender of the subject via criteria.",
+      "design": [
+        {
```

Comment:
a little confused about this 'design' collection. Right now it looks like an array of 'CriteriaFeature' but would this collection actually contain different flavors of criteria? (Preset, customSql, or criteria)?  I'm assuming the ```type``` field is the type of collection this is, and the design is a collection of criteria. I might be missing it.

Reply:
THe CohortCharacterization interface defines "design" field as object. So, depending on FeatureAnalysis type, different types of values could be put there. E.g. in case of "Preset" we just store there a name of FE parameter as string, in case of CUSTOM_FE we store there plain SQL as string, and in case of CRITERIA_SET type a collection of FeatureCriteria is put there

Reply:
Ok, thanks for the clarification.